ffi/custom_passes.cpp

```diff
@@ -719,13 +729,22 @@ struct RefPrunePass : public FunctionPass {
         CallInst *incref,
         BasicBlock *cur_node,
         SmallVectorImpl<BasicBlock*> &path_stack,
+        unsigned &subgraph_size,
+        SmallBBSet &bad_blocks,
         SmallBBSet *decref_blocks,
         SmallBBSet *raising_blocks
     ) {
         // If the current path stack exceeds the recursion depth, stop, return
         // false.
         if ( path_stack.size() >= FANOUT_RECURSE_DEPTH ) return false;
 
+        // Reject subgraph that is bigger than 1000 nodes
+        if (++subgraph_size > 1000) {
```

Comment:
Should this be configurable?

Reply:
made configurable and added test in https://github.com/numba/llvmlite/pull/644/commits/3b7b3917bbd8de50ef259ae4b6b57d5d9ec4dfd0